cognite/client/data_classes/documents.py

```diff
@@ -11,18 +11,107 @@
     CogniteResourceList,
     CogniteSort,
     EnumProperty,
+    Geometry,
     IdTransformerMixin,
     NoCaseConversionPropertyList,
 )
 from cognite.client.data_classes.aggregations import UniqueResult
 from cognite.client.data_classes.labels import Label, LabelDefinition
-from cognite.client.data_classes.shared import GeoLocation
 from cognite.client.utils._text import convert_all_keys_to_snake_case
 
 if TYPE_CHECKING:
     from cognite.client import CogniteClient
 
 
+class DocumentsGeoJsonGeometry(CogniteObject):
```

Comment:
This seems like a class that could be use outside of the documents API. Could it be called `GeometryCollection` instead?
```suggestion
class GeometryCollection(CogniteObject):
```